swift-paperless/Networking/PaperlessURLSessionDelegate.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  PaperlessURLSessionDelegate.swift
+//  swift-paperless
+//
+//  Created by Nils Witt on 24.06.24.
+//
+
+import Foundation
+import os
+
+class PaperlessURLSessionDelegate: NSObject, URLSessionTaskDelegate {
+    private var credential: URLCredential? = nil
```

Comment:
This is a public mutable, and means the class becomes non-`Sendable`.
You could make this a `let` property, and switch from setting the identity via `loadIdentityByName` to supplying it to `init` instead.